Sunrise FAQ updated changes. (#688)
graphprotocol/docs: website/pages/en/sunrise.mdx

### What are the phases of the Sunrise?

> **Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.  

Reviewer: Is the last item in the enumeration missing a verb?

Author: offers!

> **Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and simple payment options.  

Author: ```suggestion
**Sunray**: Enable support for hosted service chains, introduce a seamless upgrade flow, offer a free plan on The Graph Network, and provide simple payment options.  
```

**Sunbeam**: The upgrade window that subgraph developers will have to upgrade their hosted service subgraphs to The Graph Network. This window will end on June 12th 2024.  
**Sunrise**: Hosted service endpoints will no longer be available after June 12th 2024.

## Upgrading subgraphs to The Graph Network

### When will hosted service subgraphs no longer be available?

> Hosted service endpoints will no longer be available after June 12th, 2024.

Author: ```suggestion
Hosted service endpoints will no longer be available after June 12th, 2024. Developers will no longer be able to deploy new versions, and query endpoints will no longer be available to query.
```

2. Select the receiving wallet (the wallet that will become the owner of the subgraph).
3. Click the "Upgrade" button.

> Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it. Once you have generated an API key, you can begin making queries immediately. ([learn more](/cookbook/upgrading-a-subgraph/#what-next)).

Reviewer: ```suggestion
Once your subgraph is published, the [upgrade Indexer](#what-is-the-upgrade-indexer) will begin serving queries on it. Once you have generated an API key, you can begin making queries immediately. [Learn more](/cookbook/upgrading-a-subgraph/#what-next).
```

### How can I get support with the upgrade process?

The Graph community is here to support developers as they move to The Graph Network. Join The Graph's [Discord server](https://discord.gg/vtvv7FP) and request help in the #upgrade-decentralized-network channel.

### How can I ensure high quality of service and redundancy for subgraphs on The Graph Network?

> All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add GRT signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).

Reviewer 2: developers can add curation signal

Author: ```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add curation signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph).
```

> All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add GRT signal to subgraphs eligible for indexing rewards ([learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph)).

Reviewer: ```suggestion
All subgraphs will be supported by the upgrade indexer. In order to attract additional Indexers for redundancy and improved quality of service, developers can add GRT signal to subgraphs eligible for indexing rewards. [Learn more about adding signal to your subgraph](/publishing/publishing-a-subgraph/#adding-signal-to-your-subgraph)).
```

> Publishing requires Arbitrum ETH - upgrading your subgraph also airdrops a small amount to facilitate your first protocol interactions 🧑‍🚀

### I use a subgraph developed by someone else, how can I make sure that my service isn't interrupted?

> When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL. [Learn more](querying/querying-the-graph).

Reviewer: ```suggestion
When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL. [Learn more](/querying/querying-the-graph).
```

> When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL ([learn more](querying/querying-the-graph)).

Reviewer: ```suggestion
When the owner has upgraded their subgraph, you will be able to easily go from the subgraph's hosted service page to the corresponding subgraph on The Graph Network, and update your application to use the new subgraph's query URL. [Learn more](querying/querying-the-graph).
```

Around the start of June, Edge & Node will automatically upgrade actively queried subgraphs. This will give any third-party data consumers an opportunity to move subgraph endpoints to The Graph Network before June 12th. The subgraph owners will still be able to claim these subgraphs on the network using the hosted service upgrade flow.

> ### What happens if I don't upgrade my subgraph?

Author: This question will be updated to the following after June 12th.  (I will put in the PR request then) 

> I didn’t upgrade my subgraph by June 12th. What should I do if I still want to use it?
> 
> 
> The [hosted service](https://thegraph.com/hosted-service) homepage is still accessible and can be used to search for legacy hosted service subgraphs. If your hosted service subgraph has already been auto-upgraded, you may claim its network equivalent as the original owner. If your subgraph was not already auto-upgraded, you can still upgrade and publish it.

> Subgraphs will be quaryable on the hosted service until June 12th. Owners will still be able to upgrade their subgraphs to The Graph Network at any time, even after June 12th.

Reviewer 3: Why remove the information about deploying new versions?

Author: ```suggestion
Subgraphs will be queryable on the hosted service until June 12th, after which query endpoints will no longer be available and owners will not be able to deploy new versions. Owners will still be able to upgrade their subgraphs to The Graph Network at any time, even after June 12th.
```

No, the upgrade Indexer will only allocate the minimum amount per subgraph and will not collect indexing rewards.

> It operates on a “as needed” basis and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.

Reviewer 3: isn't "an" correct here?

Author: ```suggestion
It operates on an “as needed” basis and serves as a fallback until sufficient service quality is achieved by at least 3 other Indexers in the network for respective chains and subgraphs.
```

### How will this affect subgraph developers?

> Subgraph developers will be able to query their subgraphs on the The Graph Network almost immediately after upgrading from the hosted service or publishing from Subgraph Studio, as no lead time will be required for indexing.

Reviewer: ```suggestion
Subgraph developers will be able to query their subgraphs on The Graph Network almost immediately after upgrading from the hosted service or publishing from Subgraph Studio, as no lead time will be required for indexing.
```

Furthermore, the upgrade Indexer will stop supporting a subgraph if it has not been queried in the last 30 days.

Other Indexers are incentivized to support subgraphs with ongoing query volume. The query volume to the upgrade Indexer should trend towards zero, as it will have a small allocation size, and other Indexers will be chosen for queries ahead of it.

## About The Graph Network

### Do I need to run my own infrastructure?

No, all infrastructure is operated by independent Indexers on The Graph Network, including the upgrade Indexer ([read more below](#what-is-the-upgrade-indexer)).

> You can use the [Subgraph Studio](https://thegraph.com/studio/) to create, test, and publish your subgraph. All hosted service users must upgrade their subgraph to The Graph Network before June 12th, 2024.

Reviewer: ```suggestion
You can use [Subgraph Studio](https://thegraph.com/studio/) to create, test, and publish your subgraph. All hosted service users must upgrade their subgraph to The Graph Network before June 12th, 2024.
```

Running infrastructure for your own project is [significantly more resource intensive](/network/benefits/) when compared to using The Graph Network.

Additionally, The Graph Network is significantly more robust, reliable, and cost-efficient than anything provided by a single organization or team. Hundreds of independent Indexers around the world power The Graph Network, ensuring safety, security, and redundancy.

That being said, if you’re still interested in running a [Graph Node](https://github.com/graphprotocol/graph-node), consider joining The Graph Network [as an Indexer](https://thegraph.com/blog/how-to-become-indexer/) to earn indexing rewards and query fees by serving data on your subgraph and others.

### Should I use a centralized indexing provider?

If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, significantly [lower costs](/network/benefits/), and keeps you from being hostage at the data layer.

> With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. However, with a centralized indexing solution, the subgraph is private to the centralized provider.

Reviewer: I'm not sure the "however" works well here; to me it reads like we're going to list a bad effect of subgraphs being public.

Author: I frankly don't think the whole sentence after "however" is needed at all. It only talks about the issue with centralized indexing, but I don't think it's needed here.

Author: ```suggestion
With The Graph Network, your subgraph is public and anyone can query it openly, which increases the usage and network effects of your dapp. 
```

> Additionally, the upgrade Indexer enables 100,000 free monthly queries for your subgraph before payment is needed for additional queries.

Reviewer 3: ```suggestion
Additionally, Subgraph Studio provides 100,000 free monthly queries on the Free Plan, before payment is needed for additional usage.
```

Author: Should free plan be capitalized? 
Thank you--great catch!
